core/schema/wrapper.go

```diff
@@ -159,6 +159,9 @@ func getSelfDigest(a any) (digest.Digest, error) {
 	case *core.GitRef:
 		// FIXME can core.DigestOf(x) be used instead? When set, the TestGit/TestAuthClient test failed intermittently
 		return "", nil // fallback to using dagop ID
+	case *core.Changeset:
+		// FIXME this seems weird?
+		return "", nil
```

Comment:
Indeed :sweat_smile: 

Some discussion in private discord here: https://discord.com/channels/707636530424053791/1347633524093616190/1416099558315786451

I wonder how much of this might be removable at some point :thinking: